README.md

```diff
@@ -48,6 +48,51 @@ After that, you may run the binary as any normal program:
 ./shark-clipper-linux-x64-latest.bin
 ```
 
+#### Using the Mac Prebuilt Binary
+
+Mac requires additional security steps to run the prebuilt binary.
+These steps need to be done every time a new binary is downloaded and run for the first time.
+Running binary files is best done via the command line.
+
+This [command line primer](https://gomakethings.com/navigating-the-file-system-with-terminal/)
```

Comment:
This is not a primer, and I don't want to lead with the assumption that Mac users don't know about the terminal (some profs can get picky about that).
Say something like: "If you are not familiar with using the terminal, you can use this guide to get started.".